soc/nuvoton/npcm/Kconfig.soc

```
@@ -0,0 +1,12 @@
+# Nuvoton Cortex-M4 Embedded Controller
+
+# Copyright (c) 2024 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_NPCM
+	bool
+
+config SOC_FAMILY
+	default "nuvoton_npcm" if SOC_FAMILY_NPCM
```

```suggestion
	default "npcm" if SOC_FAMILY_NPCM
```
or
```suggestion
	default "nuvoton_npcm" if SOC_FAMILY_NUVOTON_NPCM
```
If the second then soc.yml needs update also

---

Go with npcm!
Thanks for reviewing! Resolved